prime-angular-frontend/src/app/modules/health-auth-site-reg/pages/site-management-page/site-management-page.component.html

```diff
@@ -47,7 +47,8 @@
                       [properties]="[
                             { key: 'Site Name', value: healthAuthoritySite?.siteName },
                             { key: 'Site ID', value: healthAuthoritySite?.pec },
-                            { key: 'Vendor', value: healthAuthoritySite?.healthAuthorityVendor?.vendorCode | configCode : 'vendors' }
+                            { key: 'Vendor', value: healthAuthoritySite?.healthAuthorityVendor?.vendorCode | configCode : 'vendors' },
+                            { key: healthAuthoritySite?.submittedDate ? 'Last Submitted By' : 'Last Updated By', value: getLastUpdatedUser(healthAuthoritySite?.authorizedUserName, healthAuthoritySite?.updatedTimeStamp)},
```

Comment:
Also wondering, in a shared environment like DEV or PROD, if I make a change at 4 PM Mountain Time, what will a user working in Vancouver, looking at 3:05 PM Pacific Time (five minutes later) see?  "Last Updated at ... 3:00 PM"?

Reply:
Yeah, I did think about time soon. however, not sure address it. I looked at admin event log and the time is displayed as it's. I believe it is consistent in that sense.

Reply:
Seemed to display appropriately when I was making changes in PR environment and then changing my PC's timezone settings